### PR TITLE
Information FAQ Page and other styling

### PR DESCRIPTION
**Pull request by kathytbui**
### New Feature Submissions:
* [x] Have you added an explanation of what your changes do?
* [x] Have you linted your code locally prior to submission? - one line of code is too long, but should be handled by s3 branch

### Testing:
* [x] Have you written new tests for your core changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?
* [x] All new and existing tests passed?

**Test coverage** = 100%
--------------------------------------------------------------------------------
### Change Log:
- Updated styling and fixed some errors with UI

### CODE REVIEW

File: spec/features/information/information_spec.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe 'Information page' do
+  describe 'As a visitor' do
+    it "can display information about glitch art" do
+      visit '/'
+      click_on 'What is Glitch Art?'
+      expect(current_path).to eq('/information')
+      expect(page).to have_content('What is Glitch Art?')
+      expect(page).to have_content("Have you ever accidentally opened a file in the wrong application and panicked? Little did you know you just unlocked a new databending skill! Databending is the process of manipulating a media file of a certain format in a software designed to edit files of another format. Artists have figured out how to percisely manipulate and delicately corrupt data without breaking the file.")
```
Comment (dtsehl) [inline spec/features/information/information_spec.rb:10]: Nice verbose description!

File: app/controllers/information_controller.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+class InformationController < ApplicationController
+  def index; end
```
Comment (janegreene) [inline app/controllers/information_controller.rb:2]: We could probably re-route the "about us" page to use this controller too.

**Review by dtsehl — APPROVED** [review]

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -1,14 +1,31 @@
...
+      <div class="card-body">
+        <h5 class="card-title">Instructions:</h5>
+        <p class="card-text text-left">
+        1. Enter a keyword into the search bar above and hit search.<br><br>
+        2. GlitchCraft will render two image options for glitching.<br></p>
+        <p class="card-text text-left" style="margin-left: 40px; text-left"><em>Option 1:</em> The DeepAI Image is an AI generated image from scratch based off it's perception of your keyword.<br>
+        <em>Option 2:</em> The Pixabay Image is a human tagged image relating to your keyword.<br></p>
+        <p class="card-text text-left" style="margin: 20px;">3. Select your image and cause a glitch!<br>
+        <br></p>
+        NOTE: If you do not like your image selections, simply hit the back button on your browser and create a new search!
```
Comment (janegreene) [inline app/views/users/show.html.erb:27]: I like these user instructions, nice work!

File: config/routes.rb [hunk 4]
```diff
@@ -9,8 +9,11 @@
   get '/search', to: 'search#index'
   get '/search/:id', to: 'search#show', as: :search_show
   post '/image', to: 'images#create'
+  get '/information', to: 'information#index'
+  get '/about', to: 'developers#index'
   get '/500', to: 'application#server_errors'
   get '/400', to: 'application#search_errors'
+  get '*path', to: redirect('/404'), constraints: lambda { |req|
+  req.path.exclude? 'rails/active_storage' }
```
Comment (janegreene) [inline config/routes.rb:17]: This must continue to be the last route!

File: spec/features/welcome/index_spec.rb [hunk 5]
```diff
@@ -13,7 +13,7 @@
       visit root_path
       expect(page).to have_content('GlitchCraft')
       within ('.description') do
-        expect(page).to have_content('This app allows you to glitch a photo that is created by an AI based on a word or phrase that you provide.') # Just a placeholder - Need an actual description
+        expect(page).to have_content('This app allows you to glitch a photo that is created by an AI based on a word or phrase that you provide.')
```
Comment (janegreene) [inline spec/features/welcome/index_spec.rb:16]: Do we want to specify "phrase" is really only 2 words?

**Review by janegreene — APPROVED** [review]
Wow, looking good!

**Pull request merged**